### PR TITLE
docs: add missing yaml documentation in the default order section

### PR DESCRIPTION
**Pull request by johnkrovitch**
Add missing yaml documentation in the default order section

### CODE REVIEW

File: core/default-order.md [hunk 1]
```diff
@@ -28,9 +30,20 @@ class Book
 }
 ```
 
+```yaml
+# config/api/resources/Book.yaml
+App\Entity\Book:
+    attributes:
+        order: { foo: ASC }
```
Comment (dunglas) [inline core/default-order.md:37]:
```suggestion
        order:
            foo: ASC
```

File: core/default-order.md [hunk 2]
```diff
@@ -106,3 +141,19 @@ class Book
...
 ```
+
+```yaml
+# config/api/resources/Book.yaml
+App\Entity\Book:
+    get: ~
+    get_desc_custom:
+        method: get
+        path: custom_collection_desc_foos
+        order: {name: DESC}
```
Comment (dunglas) [inline core/default-order.md:152]:
```suggestion
        order:
            name: DESC
```

File: core/default-order.md [hunk 3]
```diff
@@ -106,3 +141,19 @@ class Book
...
+    get_asc_custom:
+        method: get
+        path: custom_collection_asc_foos
+        order: {name: ASC}
```
Comment (dunglas) [inline core/default-order.md:156]:
```suggestion
        order:
            name: ASC
```

**Review by dunglas — CHANGES_REQUESTED** [review]
Awesome! I just made some minor changes to make this more YAMLy.

**Comment by alanpoulain** [comment]
Thanks @johnkrovitch.

**Pull request merged**